lib_src/src/multirate_hifi/use_vpu.h

```diff
@@ -0,0 +1,5 @@
+// Copyright 2024 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+#if defined(__XS3A__)
```

Comment:
It would be nice to have this guarded by #ifdef SRC_USE_VPU so you can choose to the scalar unit. Two reasons why:
- @xross said he is uncomfortable about changing pre-existing xcore.ai apps that have already been delivered
- Cases where every bit counts - the user might choose CPU utilisation over quality
-